backend/server/src/main/java/com/issuetracker/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.issuetracker.exception;
+
+import com.issuetracker.controller.FailedResultApi;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+    @ExceptionHandler({
+            RuntimeException.class
+    })
+    @ResponseStatus(code = HttpStatus.BAD_REQUEST)
+    public FailedResultApi<?> notAuthorizedException(RuntimeException exception) {
+        return FailedResultApi.failed(exception);
+    }
```

Comment:
```suggestion
    @ResponseStatus(code = HttpStatus.BAD_REQUEST)
    public FailedResultApi<?> notAuthorizedException(MaxLengthException exception) {
        return FailedResultApi.failed(exception);
    }
```

MaxLengthException 이 아니라, 모든 RuntimeException 에 대해서 예외처리를 하시는 이유를 알 수 있을까요?
이렇게 해버리면, 백엔드 서버측에서 잘못한 버그(가령 SQL을 잘못짰다던가...)에 대해서도 500 에러가 아니라, 400 에러가 발생을 할 텐데요.
500 에러가 아닌, 4XX 시리즈의 에러에 대해서는 서버가, 정상적으로 에러핸들링을 했다고 판단해서, 에러 로그에 남기지 않을 수도 있습니다. 그럴경우 디버깅은 더 어려워지고요.

Reply:
현재 테스트 코드 없이, 포스트맨으로 API 테스트를 하고 있다 보니, 
에러 리스폰스를 즉시 리턴 받으면 더 효율적으로 확인할 수 있다고 생각해서 설정했었습니다.
이 부분은 고치도록 하겠습니다. 지적해주셔서 감사합니다!